### ISSUE TITLE
Studies table should include a column showing the list of modalities in that study

### ISSUE THREAD

**Issue by fedorov** [ImagingDataCommons/IDC-WebApp#971]
Otherwise, the only way to figure it out is to select each of the studies, and look what modalities it includes.

If I selected in filters that I am interested in, for example, slide microscopy, but only some studies for a given case contains SM, then it is quite cumbersome to find those studies that contain the images I need.

**Comment by s-paquette** [comment]
Note that since this will be a multi-value column, it won't be sortable. (Or if we do sort it, we'll simply be sorting on a concatenated String, which might not be particularly useful to the users.)

**Comment by fedorov** [comment]
Good point, I think that's acceptable. We could also sort values within the concatenated string, and then sort the concatenated strings, which may be at least somewhat more meaningful.
